### DISCUSSION TITLE
ExplorerPatcher with TaskbarX

### DISCUSSION THREAD

**Discussion by buraCharles** [valinet/ExplorerPatcher#295] [General]
In previous versions the "Start" icon was centered with others.
In the latest "Start" version stays on the left.
Is there an explanation?

**Comment by valinet** [comment]
I haven’t made any changes there.

**Comment by buraCharles** [comment]
hello;
This is the version of TaskBarx 1.73 which is involved.
With version 1.72 and it's good.

**Reply by valinet** [reply]
TaskbarX is not my app. The developer released a new version where he upstreamed my Windows 11 compatibility fix. The version where it centers the Start button is also made by me, I played around with the code in an afternoon. The additions are not upstreamed. So yeah, if you’re using that, of course it doesn’t automagically update with my code additions as well. Use the version I provided if you want that behavior, I don’t understand why update when it is just a version bump compared to what you have, which contains basically the same things as the released version, plus that behavior you describe. 

So yeah, definitely not an issue with EP.

**Comment by buraCharles** [comment]
Thank you for these explanations
The first time I used TaskBarx it was your compil. And it worked.
So I adopted your TaskBarX version again.

![taskbar](https://user-images.githubusercontent.com/93889254/140774385-13ce7c8b-9ed7-4fcf-9235-1f0eaff0bcdc.jpg)
